Safari FIDO U2F Extension/SafariExtensionHandler.swift

```diff
@@ -5,6 +5,25 @@
 //  Created by Yikai Zhao on 10/13/16.
 //  Copyright © 2016 Yikai Zhao. All rights reserved.
 //
+//  Permission is hereby granted, free of charge, to any person obtaining
```

Comment:
No need to replicate the license. You can do this the way Facebook's React does this:

```
/**
 * Copyright (c) 2013-present, Facebook, Inc.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 */
```